test/plugin_helper/test_child_process.rb

```diff
@@ -529,19 +532,25 @@ def configure(conf)
       sleep TEST_WAIT_INTERVAL_FOR_BLOCK_RUNNING until m.locked? || ran
       m.lock
       assert_equal Encoding.find('utf-8'), str.encoding
-      expected = "\xEF\xBF\xBD\xEF\xBF\xBD\x00\xEF\xBF\xBD\xEF\xBF\xBD".force_encoding("utf-8")
+      replacement = "\uFFFD" # U+FFFD (REPLACEMENT CHARATER)
```

Comment:
```suggestion
      replacement = "\uFFFD" # U+FFFD (REPLACEMENT CHARACTER)
```

Reply:
Ooops...

Reply:
Fixed in a2c4b4640eea836730957c9946a39f946d613329